<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createsession</class>
 <widget class="QWidget" name="createsession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session Editor</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame { border: 1px solid gray; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="lbl_traffic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Session Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_edit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {border: 1px solid gray};</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QRadioButton" name="rb_eth">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ethernet II</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>374</width>
       <height>66</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QGroupBox{border: 0px solid gray;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_mac_src">
        <property name="styleSheet">
         <string notr="true">border: 0px;</string>
        </property>
        <property name="text">
         <string>MAC Source:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_mac_source">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
        <property name="inputMask">
         <string>HH:HH:HH:HH:HH:HH</string>
        </property>
        <property name="text">
         <string>00:0a:95:9d:68:16</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="mac_checkBox">
        <property name="text">
         <string>Override MAC Source</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_mac_dest">
        <property name="styleSheet">
         <string notr="true">border: 0px;</string>
        </property>
        <property name="text">
         <string>MAC Destination:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_mac_dest">
        <property name="inputMask">
         <string>HH:HH:HH:HH:HH:HH</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>641</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{border: 1px solid gray;}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="payload_checkBox">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="payloadEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>641</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{border: 1px solid gray; border-bottom: none}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QRadioButton" name="rb_ip4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>45</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>IPv4</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>10</y>
       <width>581</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QGroupBox{border: 0px solid gray;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_16">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Increment Count :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="txt_ip_dest">
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
        <property name="inputMask">
         <string>000.000.000.000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="overrideipsrc">
        <property name="text">
         <string>Override IP Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="txt_ipincrement">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_ipsrc">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>IP Source:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_ipsrcincrement">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Increment Count :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="txt_ip_src">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
        <property name="inputMask">
         <string>000.000.000.000</string>
        </property>
        <property name="text">
         <string>192.168.1.1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_ttl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>TTL :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>IP Destination:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="ttl_checkBox">
        <property name="text">
         <string>Override TTL</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="txt_ttl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>64</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>106</y>
       <width>600</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QGroupBox{border: 0px solid gray;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="lb_ip6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>48</y>
       <width>45</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>IPv6</string>
     </property>
    </widget>
    <zorder>groupBox_5</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>rb_ip4</zorder>
    <zorder>lb_ip6</zorder>
   </widget>
   <widget class="QFrame" name="frame_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>279</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{border: 1px solid gray; border-bottom: none;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="rb_tcp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_udp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_icmp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ICMP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_sctp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SCTP</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>10</y>
       <width>561</width>
       <height>118</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QGroupBox{border: 0px solid gray;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_tcp_src">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Source Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_tcp_src_port">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_src_porttcp">
        <property name="text">
         <string>Override Source Port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>RST</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkBox_ack_tcp">
        <property name="text">
         <string>Override Ack</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>FIN</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>PSH</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>URG</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_dest_tcp"/>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>ACK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_urgent">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Urgent Pointer:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLineEdit" name="txt_urgent">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBox_seq_tcp">
        <property name="text">
         <string>Override Seq</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Destination Port:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txt_tcp_ack_no">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txt_seq_tcp">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_tcp_ack">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ack no:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_seq_tcp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Sequence no:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>SYN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>10</y>
       <width>370</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QGroupBox{border: 0px solid gray;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_udp_src_port">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="text">
         <string>80</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="sourceport_checkBox">
        <property name="text">
         <string>Override Source Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_src_port_udp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Source Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_udp_dst">
        <property name="text">
         <string>Destination Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_udp_dest_port">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>80</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{border: 0px solid gray;}</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="cancel_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>1448916123_plus_16.png</normaloff>1448916123_plus_16.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_Button">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>1448918292_tick_16.png</normaloff>1448918292_tick_16.png</iconset>
     </property>
    </widget>
   </widget>
   <zorder>frame1</zorder>
   <zorder>frame_4</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame_31</zorder>
   <zorder>frame_21</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
